auth/user_mgt.go

```diff
@@ -400,12 +416,26 @@ func (u *UserToCreate) preparePayload() (map[string]interface{}, error) {
 			if err := validate(v); err != nil {
 				return nil, err
 			}
+			reflect.ValueOf(user).Elem().FieldByName(strings.Title(key)).SetString(params[key].(string))
 		}
 	}
+	if params["disabled"] != nil {
+		user.Disabled = params["disabled"].(bool)
+		if !user.Disabled {
+			user.ForceSendFields = append(user.ForceSendFields, "Disabled")
+		}
+	}
+	if params["emailVerified"] != nil {
+		user.EmailVerified = params["emailVerified"].(bool)
+		if !user.EmailVerified {
+			user.ForceSendFields = append(user.ForceSendFields, "EmailVerified")
+		}
+	}
+
 	return params, nil
 }
 
-func (u *UserToUpdate) preparePayload() (map[string]interface{}, error) {
+func (u *UserToUpdate) preparePayload(user *identitytoolkit.IdentitytoolkitRelyingpartySetAccountInfoRequest) (map[string]interface{}, error) {
```

Comment:
The first is always ignored, might as well omit it here. (and in the parallel function)

Reply:
Done